modules/aws_k8s_service/aws-k8s-service.yaml

```diff
@@ -212,6 +215,9 @@ inputs:
     description: The max amount of helm revisions to keep track of (0 for infinite)
     default: 25
 extra_validators:
+  resources:
+    cpu: any(int(min=1), regex('^[1-9][0-9]{0,3}$'), required=True)
```

Comment:
I think the regex could very well be {0,4}, maybe someone wants to assign 10 or more cores for a really CPU intensive batch process or so?